### DISCUSSION TITLE
SceneryStack Show and Tell: Paschen's Law Sim

### DISCUSSION THREAD

**Discussion by dave-english** [scenerystack/community#57] [Show and tell]
[Paschen's Law Sim](https://www.youtube.com/watch?v=1fzV__itkhQ) (YouTube link)
Author: David English
Science Advisor: Dr Oak Nelson
Many thanks to the SceneryStack community

The power of lightning has sparked public imagination since the time of Ben Franklin.  Electric arc formation involves a phase change from gas to plasma, a subject of intense contemporary research as teams worldwide race toward the goal of harnessing plasma to create green energy from controlled nuclear fusion.

This Paschen Sim is the starting point of a collection of resources to introduce high school and college students to plasma and fusion science. These fields are expected to grow exponentially in the coming decades, as we’ll need to train workers to build and operate a fleet of nuclear fusion reactors. These subjects are currently under-represented in high school and even many undergraduate level science curricula.

Friedrich Paschen was a German physicist who measured the minimum voltage required to pass an electric arc through gas at different pressures, while changing the electrode spacing. Widely spaced electrodes require dangerously high voltages to pass an arc. Perhaps more surprisingly, at very low pressure there is also a minimum separation required to pass an arc, such that at close enough spacing, the electrical arc may not even travel along the shortest path!

Paschen published a mathematical equation relating these variables in 1889, a model now known as Paschen’s law. About a decade later the British physicist JS Townsend provided a theoretical explanation for this relationship.

Paschen Sim uses slider bars to set the separation between electrodes, voltage, and gas pressure. The student selects the type of gas (one of five choices) from a combo box. A red marker appears over a graphic of Paschen’s curve, allowing the student to follow where they are on the curve. A simulated electric arc and emission glow appear between the electrodes when conditions are right.

Future project goals include
- Optimizing screen layout
- Replacing sliders with simulated lab equipment
- Adding a second screen view to the model, to explore the related Townsend theory
- Exploring ways to make the sim more compliant with SceneryStack and PhET best-practices.
- Developing a full suite of tools and educational programs to introduce plasma science at an earlier grade level than it is currently taught.

I’ve gotten encouragement and advice from a lot of other people to begin this project. Dr Nelson introduced me to Paschen’s experiment, taught me to set up a github repository, and has given me a lot of encouragement along the way. I have also received help and advice from the SceneryStack community including Brett Fiedler, Jonathan Olson, Sam Reid and Chris Malley. 

Regarding my own background: I teach high school level chemistry and other physical sciences, and coach a Science Olympiad team (2024 NJ state champions, 4th in nation). I have a Master’s degree in educational technology, and attended the 2019 ISTE conference. As an amateur programmer I’ve published several independent games in Javascript-related languages under the pseudonym Doug Egan. Paschen Sim has been my introduction to Typescript, Github, and the SceneryStack library. My development environment is VSCode installed in the Linux partition of a Chromebook, a device widely used in public schools, which might be of interest to other beginning developers.

I wrote the code for this simulation during the months of July and August 2024, while on summer holiday. I’m back to school now and I won’t have much time to continue this Paschen Sim project until next summer. I am hoping by then to acquire some grants or other funding to complete the Sim, develop lesson plans, then introduce it to students and other teachers. I am open to other new projects in educational software development and promotion as well.

If you'd like to download the code from github, or join me as a collaborator, please message me at dougegan2@gmail.com

**Comment by nelsonand** [comment]
Great work @dave-english !
